evalml/pipelines/components/estimators/regressors/catboost_regressor.py

```diff
@@ -113,9 +114,25 @@ def fit(self, X, y=None):
         self.input_feature_names = list(X.columns)
         X, y = super()._manage_woodwork(X, y)
         X = downcast_int_nullable_to_double(X)
+
+        X = handle_float_categories_for_catboost(X)
         self._component_obj.fit(X, y, silent=True, cat_features=cat_cols)
         return self
 
+    def predict(self, X):
```

Comment:
Figured if I was going to handle categories for the regressor (and I did double check it was necessary for `predict`!) I'd have to add an explicit predict method -- let me know if there's a better way that I missed though!

Reply:
looks good!